lms/templates/header.html

```diff
@@ -6,7 +6,6 @@
 from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext as _
 
-from openedx.core.djangolib.markup import HTML, Text
```

Comment:
This theme was forked from master, but the djangolib.markup libaray isn't present in Dogwood